docs/player-setup.md

```diff
@@ -0,0 +1,87 @@
+# Player Setup
+Player setup requires the following steps to create a player instance:
+### Step 1 - Load the Library
+```html
+<script src="//PATH/TO/PLAYER/LIB/FILENAME.js"></script>
+```
+### Step 2 - Create the Player Container
+```html
+<div id="player-container"></div>
+```
+### Step 3 - Define Your Configuration
+```js
+var config = {
+	targetId: "player-container",
+	player: { ... }, // player configuration
+	provider: { // provider configuration
+		...
+		partnerId: YOUR_PARTNER_ID
+		...
+	},
+	ui: { ... } // ui configuration
+};
+```
+
+The following sections are examples of common (and important) configurations for the player setup. 
+
+#### Example: Using a Kaltura Session (KS)
+If you need to use a KS for your media requests, configure it inside your provider configuration:
+```js
+var config = {
+	...
+	provider: { 
+		...
+		partnerId: YOUR_PARTNER_ID,
+		ks: 'YOUR_KS'
+		...
+	}
+	...
+};
+var player = KalturaPlayer.setup(config);
+``` 
+
```

Comment:
Link to KS docs: https://vpaas.kaltura.com/documentation/VPaaS-API-Getting-Started/how-to-create-kaltura-session.html

Reply:
@OrenMe good addition!